### ISSUE TITLE
Does this library support ERC-20?

### ISSUE THREAD

**Issue by remembermeLiu** [ethers-io/ethers.objc#31]

**Comment by ricmoo** [comment]
Absolutely. Just use the Contract object, and give it the ERC-20 ABI, along with your token address and a provider (or a signer, for managing the asset).

An ERC-20 token is just a normal contract, nothing special is required to interact with one. The `utils.parseUnits` and `utils.formatUnits` also make it easy to convert between user input to safe values and from safe values to display logic. :)

**Comment by ricmoo** [comment]
Oh! So sorry, I thought this was for the ethers.js library. I didn’t notice the repo you posted to.

This library does not easily handle ERC-20. Sorry. :s
